prometheus-rules/prometheus-vmware-rules/Chart.yaml

```
@@ -1,7 +1,7 @@
 apiVersion: v2
 description: A collection of Prometheus alert rules.
 name: prometheus-vmware-rules
-version: 1.0.9
+version: 1.0.10
```

```suggestion
version: 1.1.0
```

Minor please, not patch. This change is quite invasive.